android/src/main/AndroidManifest.xml

```diff
@@ -1,3 +1,20 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-          package="com.bleplx">
+          package="com.bleplx"
+          xmlns:tools="http://schemas.android.com/tools">
+
+  <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
+  <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
```

Comment:
Currently, you have 'never for location' flag and two location permissions maybe 'android:maxSdkVersion="30"'